kratos/solving_strategies/strategies/solving_strategy.h

```diff
@@ -109,16 +109,27 @@ class SolvingStrategy
     ///@name Life Cycle
     ///@{
 
-    /** Constructor.
+    /**
+     * @brief Default constructor. (with parameters)
+     * @param rModelPart The model part of the problem
+     * @param ThisParameters The configuration parameters
+     */
+    explicit SolvingStrategy(ModelPart& rModelPart, Parameters ThisParameters)
+        : mrModelPart(rModelPart)
+    {
+        const bool move_mesh_flag = ThisParameters.Has("move_mesh_flag") ? ThisParameters["move_mesh_flag"].GetBool() : false;
```

Comment:
missing validate

Reply:
There can not be validate , look #3185, the base class constructor is called in the derived class, so it can not validate or it will crash

Reply:
That's why HAs is used instead